gfp.go

```diff
@@ -16,6 +19,14 @@ func newGFp(x int64) (out *gfP) {
 	return out
 }
 
+func fromBytes(b [32]byte) *gfP {
```

Comment:
The draft also defines a method for hashing messages to elements of a field. See Section 5.

Reply:
I've changed it accordingly.